code/game/gamemodes/cult/cult_items.dm

```diff
@@ -45,11 +45,13 @@
 		else
 			user.adjustBruteLoss(rand(force/2, force))
 		return
-	if(!IS_CULTIST(target))
-		var/datum/status_effect/cult_stun_mark/S = target.has_status_effect(STATUS_EFFECT_CULT_STUN)
-		if(S)
-			S.trigger()
-	..()
+	. = ..()
+	if(IS_CULTIST(target) || . == FALSE)
```

Comment:
```suggestion
	if(IS_CULTIST(target) || !.)
```
Parent can return `null`, which is not equivalent to FALSE.

Reply:
`!` is just a more robust check yeah, `== FALSE` should almost never be done

Reply:
`. == FALSE` required because it will be `FALSE` if attack was blocked. Otherwise it would be `null`, thats why i check here for that. I wish that to return something better than `null` and `FALSE` but we have what we have

Reply:
The only way (hopefully) that the parent `attack()` will return `null` in this case is if the attacker is pacified, which `. == FALSE` will let go through and detonate the mark even without playing an animation or sound.